src/components/agent/services.ts

```diff
@@ -142,4 +151,26 @@ export class AgentServices {
 
     return !!botFlow;
   }
+
+  static async getSupportNumbers(): Promise<SupportResponse> {
+    const agent = await UserRootDAO.getAgent();
+
+    if (!agent) throw new CustomError(ERR.AGENT_UNSET);
+
+    return {
```

Comment:
**suggestion:** Improved payment handling, but review error cases

The new releasePayment method is a good improvement over the old markAsPaid, providing more comprehensive handling of the payment process. However, please review the error handling to ensure all possible error cases are accounted for, especially when interacting with external services.